elements/etherswitch/etherswitch.cc

```diff
@@ -38,24 +40,69 @@ EtherSwitch::~EtherSwitch()
 int
 EtherSwitch::configure(Vector<String> &conf, ErrorHandler *errh)
 {
-    return Args(conf, this, errh)
+    if (Args(conf, this, errh)
 	.read("TIMEOUT", SecondsArg(), _timeout)
-	.complete();
+	.complete() < 0)
+        return errh->error("bad timeout");
```

Comment:
Just return -1. The error could be that there is an unknown argument, not bad timeout.